packages/admin-panel/src/table/columnTypes/columnFilters.js

```diff
@@ -10,15 +10,27 @@ import PropTypes from 'prop-types';
  * https://github.com/tannerlinsley/react-table/tree/v6/examples/custom-filtering
  */
 
-const options = [
-  { label: 'Show All', value: '' },
-  { label: 'Yes', value: true },
-  { label: 'No', value: false },
-];
-
 export const BooleanSelectFilter = ({ filter, onChange }) => (
   <Select
-    options={options}
+    options={[
+      { label: 'Show All', value: '' },
+      { label: 'Yes', value: true },
+      { label: 'No', value: false },
+    ]}
+    onChange={event => onChange(event.target.value)}
+    value={filter ? filter.value : ''}
+  />
+);
+
+// eslint-disable-next-line react/prop-types
```

Comment:
```suggestion
```
We can remove this eslint-disabler now that you have added the prop types. Thanks for adding those.